receiver/awscontainerinsightreceiver/config.go

```diff
@@ -57,4 +57,8 @@ type Config struct {
 	// EnableControlPlaneMetrics enables additional metrics sourced from the Kubernetes API server /metrics prometheus endpoint
 	// The default value is false.
 	EnableControlPlaneMetrics bool `mapstructure:"enable_control_plane_metrics"`
+
+	// EnableGpuMetric toggles GPU monitoring where metrics are scraped from vendor specific sources
+	// The default value is true meaning GPU metrics get collected out of the box unless it's disabled
+	EnableGpuMetric bool `mapstructure:"gpu_metrics"`
```

Comment:
If you want the default value to be true, you'll need to update the container insights receiver's factory so that the default configuration sets it to true.

Reply:
it will be set by the agent